app/services/redis_storage.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class RedisStorage
+  class << self
+    def [](key)
+      Redis.current.get(key)
+    end
+
+    def []=(key, value)
+      Redis.current.set(key, value)
+    end
+
+    # default options = {
+    #   block:    1   - seconds how long to wait for the lock to be released
+    #             0   - return false immediately
+    #   sleep:    0.1 - seconds how long the polling interval should be when :block is given.
+    #   expire:   10  - seconds when the lock should be considered stale when something went wrong
+    # }
+    def lock!(key, options = { block: 1, sleep: 0.1, expire: 10 })
```

Comment:
Is `block` duration enough?

Reply:
We'll increase it if needed.

Reply:
Ok, in this case could you please change error log to rollbar notification?
here: https://github.com/CleverLabs/deployqa/pull/271/files#diff-6d9f16a18aa01eee0c66f1b451e96ee7R30

To something like:
```ruby
Rollbar.info("message")
```

Reply:
ok

Reply:
Done